jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferAccumulator.java

```diff
@@ -24,26 +24,26 @@
 /**
  * Accumulates data into a list of ByteBuffers which can then be combined into a single buffer or written to an OutputStream.
  * The buffer list automatically grows as data is written to it, the buffers are taken from the
- * supplied {@link ByteBufferPool} or freshly allocated if one is not supplied.
+ * supplied {@link RetainableByteBufferPool} or freshly allocated if one is not supplied.
  *
  * The method {@link #ensureBuffer(int, int)} is used to write directly to the last buffer stored in the buffer list,
  * if there is less than a certain amount of space available in that buffer then a new one will be allocated and returned instead.
  * @see #ensureBuffer(int, int)
  */
 public class ByteBufferAccumulator implements AutoCloseable
```

Comment:
This class is share a bit of functionality, but more visibly its name with `RetainableByteBufferPool.Accumulator`, which I find confusing. It feels like they should be merged, or named differently.